challenge/dns01/nameserver.go

```diff
@@ -229,15 +231,39 @@ func dnsQuery(fqdn string, rtype uint16, nameservers []string, recursive bool) (
 	m := createDNSMsg(fqdn, rtype, recursive)
 
 	var in *dns.Msg
-	var err error
+	var errG error
```

Comment:
What does `errG` stand for? Neither "backoff", nor "retry", nor "query" contain a "g".

Reply:
https://github.com/go-acme/lego/pull/1180/files#diff-2064c998dfa170d0767a3036f371c862R262

Reply:
That doesn't explain the "G" in the variable name. I can see where it is used/assigned, but I wonder why it's named `errG` and not something like `errQuery` or `errBackoff`...

Reply:
`G` because `Global`, but you are right I can improve this name.

Reply:
Ah, of course. That I did not think of. Thanks for clarification/correction :+1: